__tests__/distributors/adoptium-installer.test.ts

```diff
@@ -0,0 +1,164 @@
+import { HttpClient } from '@actions/http-client';
+
+import {
+  AdoptiumDistribution,
+  AdoptiumImplementation
+} from '../../src/distributions/adoptium/installer';
```

Comment:
../../src/distributions/adoptium/installer should be changed to ../../src/distributions/temurin/installer
(and src directory name changed accordingly as well).